### DISCUSSION TITLE
graphql-java-extended-scalars version numbering

### DISCUSSION THREAD

**Discussion by mattTea** [graphql-java/graphql-java#3106] [General]
Are version numbers for the graphql-java-extended-scalars library of format 2023-01-24T02-11-56-babda5f stable releases?

I can see that these are shown on [Maven](https://mvnrepository.com/artifact/com.graphql-java/graphql-java-extended-scalars/2023-01-24T02-11-56-babda5f) as releases with what appears to be no difference in labelling/tagging to releases of format [20.0](https://mvnrepository.com/artifact/com.graphql-java/graphql-java-extended-scalars/20.0), but the above looks more like a snapshot release to me.

I ask as our dependency tooling ([Renovate](https://github.com/renovatebot/renovate)) is picking up version 2023-01-24T02-11-56-babda5f from Maven instead of 20.0 (which appears to be the latest stable release here in GitHub)

Thanks

**Comment by dondonz** [comment]
Hi @mattTea, the releases with date stamps in them are master build releases, and the versions lining up with graphql-java versions such as 20.0 are the proper, stable releases.

For Renovate, is there a way to tag the stable releases so the master build ones are no longer picked up by the tool?

**Reply by mattTea** [reply]
Thanks @dondonz
We're currently using a pretty nasty regex to force the use of v20.0 instead of the master build release above, but the Renovate maintainers have suggested a better option in this [discussion](https://github.com/renovatebot/renovate/discussions/20771) now too

**Reply by dondonz** [reply]
Thanks for asking the Renovate team! I'll have a look at what we can do for versioning so we can avoid special Renovate rules (or nasty regexes!)

**Reply by dondonz** [reply]
I've changed the extended scalars build versions to start with `0.0.0` https://github.com/graphql-java/graphql-java-extended-scalars/pull/92

Hopefully that will stop Renovate from picking up build versions, as they won't appear as the "latest" version, when sorting versions in descending order.

**Reply by mattTea** [reply]
Great! Thanks again @dondonz
